patterns/atoms/media/video/video.twig

```diff
@@ -0,0 +1,4 @@
+<video width="{{video.width}}" height="{{video.height}}" controls="{{video.controls}}" class="{{video.class}}">
```

Comment:
that goes against the current structure that pattern.yml file has been going with - where attributes for a given element are one-tier down (see ...atoms/media/video/pattern.yml). This would mean changing the structure to:

```
data:
  video:
    attr:
      class: base__video
      src: http://media.w3.org/2010/05/bunny/movie.ogv
      width: 640
      height: 480
      controls: controls
```